lib/Zadm/Schema/Bhyve.pm

```diff
@@ -127,16 +127,58 @@ $SCHEMA = sub($self) {
         array       => 1,
         description => 'network interface',
         members     => {
+            netif => {
+                optional    => 1,
+                description => 'network interface type',
+                validator   => $self->sv->elemOf(qw(virtio virtio-net-viona virtio-net e1000)),
+                'x-netprop' => 1,
+            },
             feature_mask => {
                 optional    => 1,
-                description => 'bhyve viona feature mask',
+                description => '(viona) set negotiated feature mask',
                 validator   => $self->sv->regexp(qr/^(?:0x[[:xdigit:]]+|\d+)$/i, 'expected a numeric value'),
                 'x-netprop' => 1,
             },
-            netif => {
+            vqsize => {
                 optional    => 1,
-                description => 'network interface type',
-                validator   => $self->sv->elemOf(qw(virtio virtio-net-viona virtio-net e1000)),
+                description => '(viona) set ring size - power of 2 between 4 and 32768 (default: 1024)',
+                validator   => $self->sv->elemOf(map { 2 ** $_ } 2..15),
+                'x-netprop' => 1,
+            },
+            mtu => {
+                optional    => 1,
+                description => '(virtio) set mtu',
+                validator   => $self->sv->numRange(60, 65535),
+                'x-netprop' => 1,
+            },
+            backend => {
+                optional    => 1,
+                description => '(virtio,e1000) select backend network interface',
```

Comment:
As per IRC wasn't immediately clear that the (virtio,e1000) meant it only applies for those types, although I don't have a better suggestions :(

Maybe `select backend network interface (applies to: virtio,e1000) ` ?